source/adapters/level_zero/command_buffer.cpp

```diff
@@ -86,14 +86,41 @@ graphs.
 
 */
 
+namespace {
+/// Checks the version of the level-zero driver.
+/// @param Context execution context
+/// @param VersionMajor Major verion number to compare to.
+/// @param VersionMinor Minor verion number to compare to.
+/// @param VersionBuild Build verion number to compare to.
+/// @return true is the version of the driver is higher than the compared
+/// version
+bool check_driver_version(ur_context_handle_t Context, uint32_t VersionMajor,
```

Comment:
Don't think snake_case is used for function names
```suggestion
bool CheckDriverVersion(ur_context_handle_t Context, uint32_t VersionMajor,
```